.github/workflows/acceptance.yml

```diff
@@ -106,9 +106,9 @@ jobs:
         uses: actions/cache@v2
         with:
           path: ${{ steps.pip.outputs.pip-cache-dir }}
-          key: ${{ runner.os }}-pip-py${{ steps.python-version.outputs.python-version }}-${{ hashFiles('**/requirements-*.txt') }}
+          key: ${{ runner.os }}-pip-py${{ steps.python-version.outputs.python-version }}-${{ steps.pip.outputs.pip-version }}-${{ hashFiles('**/requirements-*.txt') }}
```

Comment:
Adding `pip-version` permits the cache to be invalidated when we have a new pip version.
I was not able to reproduce the issues described in [this PR](https://github.com/getsentry/sentry/pull/23775#issue-571357017) until there would be a cache miss.